config/initializers/airbrake.rb

```diff
@@ -1,8 +1,10 @@
 if ENV['ERRBIT_API_KEY'].present?
+  errbit_uri = Plek.find('errbit')
+
   Airbrake.configure do |config|
-    config.api_key = ENV['ERRBIT_API_KEY']
-    config.host = "errbit.#{ENV['GOVUK_APP_DOMAIN']}"
-    config.secure = true
-    config.environment_name = ENV['ERRBIT_ENVIRONMENT_NAME']
+    config.project_key = ENV['ERRBIT_API_KEY']
+    config.project_id = 1 # dummy, not used in Errbit
+    config.host = errbit_uri
```

Comment:
This is the same code that we've used for all the various Rails 5 upgrades, eg https://github.com/alphagov/publishing-api/blob/master/config/initializers/airbrake.rb, so I'm confident it works.